staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_progress_test.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cacher
+
+import (
+	"context"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"k8s.io/klog/v2"
+	"k8s.io/klog/v2/ktesting"
+	"k8s.io/utils/clock"
+	testingclock "k8s.io/utils/clock/testing"
+)
+
+var (
+	pollPeriod      = time.Millisecond
+	minimalNoChange = 20 * time.Millisecond
+	pollTimeout     = 100 * time.Millisecond
```

Comment:
Let's make it bigger - 100ms is too low for heavily overloaded environments.

Given we're effectively never exercising pollTimeout anywhere (unless something actually goes wrong) let's make it 5s.

Reply:
Done